### ISSUE TITLE
A question about the results of Open-vocabulary COCO

### ISSUE THREAD

**Issue by SCUTjinchengli** [hanoonaR/object-centric-ovd#10]
Thanks for your impressive work!
I have a question, that is, when conducting experiments on COCO, the github's results are inconsistent with the paper's results. 
Could you please give some explanations about it?

**Comment by hanoonaR** [comment]
@SCUTjinchengli,

Thank you for showing your interest in our work. The numbers reported in the README are now consistent with the numbers on the [paper](https://arxiv.org/pdf/2207.03482.pdf) (latest version on arXiv). Kindly point us to what exactly you found a mismatch, so we may correct it. 

The numbers that are currently reported are on our OVD model trained with weak pseudo labels from pre-trained MViT that we train on filtered LMDet (‡LMDet) dataset by entirely restricting any exposure to the novel/rare classes in evaluation (more details in paper). This is the reason why we have updated the numbers on the paper and this repository. 

However, we have not yet uploaded the latest model checkpoints, which we will release soon. 

Please let us know if this helps. Thank you.

**Comment by SCUTjinchengli** [comment]
@hanoonaR 
Thanks for your reply!
There are some mismatch results. For example, the reported AP_novel of [COCO_OVD_RKD_PIS_WeightTransfer](https://github.com/hanoonaR/object-centric-ovd/blob/main/configs/coco/COCO_OVD_RKD_PIS_WeightTransfer.yaml) is 36.6 while the reported AP_novel in the paper (arxiv-v1) is 40.3.

After you updated the results (that is, you check the LMDet again, remove the corresponding data, and train the pre-trained model again, right?), the new SOTA on COCO over RN50-C4 under OVD setting is 36.6, right?

By the way, we are still unable to download a new version of arxiv, i.e., arxiv-v2.

Thanks you.

**Comment by hanoonaR** [comment]
@SCUTjinchengli,

Yes, you are right. To avoid any leakage of supervision in the pre-training stage, we re-train the MViT model on the filtered data. 

However, paper is updated on arXiv as well. Kindly check the link: https://arxiv.org/pdf/2207.03482.pdf 

Kindly verify, thank you.

**Comment by SCUTjinchengli** [comment]
![image](https://user-images.githubusercontent.com/45969813/196120268-0b6017d8-3b95-4731-90ff-ac84213183fe.png)

From this link, as we can see, the result is 40.3 but not 36.6. Maybe there are some bugs in arxiv.....

**Comment by SCUTjinchengli** [comment]
@hanoonaR 

Thanks for your reply! We can download the updated paper.
Have a nice day!
